test/ecto/query/builder_test.exs

```diff
@@ -124,6 +124,13 @@ defmodule Ecto.Query.BuilderTest do
            escape(quote(do: ^([] ++ [])), [], __ENV__)
   end
 
+  test "escape_binding with named binding in the input" do
```

Comment:
This is a fragile test too, we don't need to check the generated AST on expansion.